Blizzard.IO.RabbitMQ.Tests/Rpc/NetqRabbitRpcServerTests.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Blizzard.IO.RabbitMQ.Rpc;
+using EasyNetQ;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Tests.Rpc
+{
+    class RequestStub
+    {
+        public int Value { get; set; }
+    }
+
+    class RespondStub
+    {
+        public int Value { get; set; }
+    }
+
+    public class NetqRabbitRpcServerTests
+    {
+        private NetqRabbitRpcServer _server;
+        private Mock<INetqRpcRabbitConnection<Func<Type, object>>> _netqRpcRabbitConnectionMock;
+        private Mock<IBus> _busMock;
+
+        [SetUp]
+        public void Setup()
+        {
+            _busMock = new Mock<IBus>();
+            _netqRpcRabbitConnectionMock = new Mock<INetqRpcRabbitConnection<Func<Type, object>>>();
+
+            _netqRpcRabbitConnectionMock.Setup(connection => connection.Bus).Returns(_busMock.Object);
+            Mock<ILogger> loggerMock = new Mock<ILogger>();
+            Mock<ILoggerFactory> lfMock = new Mock<ILoggerFactory>();
+            lfMock.Setup(lf => lf.CreateLogger(It.IsAny<string>())).Returns(loggerMock.Object);
```

Comment:
Arrange all the mock initialization and then all the setups

Reply:
I did something difrenet cause the logger is not a field so i wanted to separate them